pkg/controller/windowsmachine/windowsmachine_controller.go

```diff
@@ -69,23 +70,23 @@ func newReconciler(mgr manager.Manager, networkConfig clusternetwork.ClusterNetw
 		return nil, errors.Wrap(err, "error creating kubernetes clientset")
 	}
 
-	signer, err := signer.Create()
+	serviceCIDR, err := networkConfig.GetServiceCIDR()
 	if err != nil {
-		return nil, errors.Wrapf(err, "error creating signer using private key: %v", wkl.PrivateKeyPath)
+		return nil, errors.Wrap(err, "error getting service CIDR")
 	}
 
-	serviceCIDR, err := networkConfig.GetServiceCIDR()
+	namespace, err := k8sutil.GetWatchNamespace()
 	if err != nil {
-		return nil, errors.Wrap(err, "error getting service CIDR")
+		return nil, errors.Wrap(err, "failed to get watch namespace")
 	}
 
 	return &ReconcileWindowsMachine{client: client,
 			scheme:             mgr.GetScheme(),
 			k8sclientset:       clientset,
 			clusterServiceCIDR: serviceCIDR,
-			signer:             signer,
 			vxlanPort:          networkConfig.VXLANPort(),
 			recorder:           mgr.GetEventRecorderFor(ControllerName),
+			opNamespace:        namespace,
```

Comment:
`opNamespace` is not appropriately named given it is the namespace the operator is watching. In our case it just so happens to be the same as the namespace the operator is running in.